bluesky/suspenders.py

```diff
@@ -533,7 +533,9 @@ class SuspendWhenChanged(SuspenderBase):
     
     Only resume if allowed AND when monitored equals expected.
     
-    .. note: this suspender is designed to require bluesky restart if value changes
+    .. note:
+
+       This suspender is designed to require bluesky restart if value changes
```

Comment:
```suggestion
       This suspender is designed to require bluesky restart if value changes.
```

Reply:
Left a minor suggestion to have this block as a complete sentence.

Reply:
done